core-bundle/src/Migration/AbstractMigration.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Migration;
+
+abstract class AbstractMigration implements MigrationInterface
+{
+    public function getName(): string
+    {
+        return str_replace('\\', ' ', static::class);
```

Comment:
The default name is `Terminal42 FooBarBundle Migration SomeMigration`?

Reply:
Yes. Can you think of a better default name that identifies the migration unambiguously?

Reply:
Maybe just the class name, then its obvious there‘s no name 🤷‍♂️